Xamarin.Forms.Controls/GalleryPages/VisualStateManagerGalleries/StateTriggersDirectlyOnElements.xaml

```diff
@@ -0,0 +1,38 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<ContentPage
+    xmlns="http://xamarin.com/schemas/2014/forms"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    x:Class="Xamarin.Forms.Controls.GalleryPages.VisualStateManagerGalleries.StateTriggersDirectlyOnElements"
+    Title="StateTriggers directly on Elements">
+    <ContentPage.Content>
+        <Grid>
+            <VisualStateManager.VisualStateGroups>
+                <VisualStateGroup>
+                    <VisualState
+                        x:Name="Landscape">
+                        <VisualState.StateTriggers>
+                            <StateTrigger IsActive="{Binding ToggleStateInverse}" />
```

Comment:
I updated this sample to use StateTrigger with a binding and it doesn't seem like the bindingcontext is propagating.

The BindingContext of the Grid should propagate to the state triggers.

Here's an example from UWP
https://github.com/microsoft/Windows-universal-samples/blob/master/Samples/XamlStateTriggers/cs/ItemTemplateControl.xaml#L37

Using a similar concept

Reply:
Good one. Fixed and included more samples using Bindings with StateTriggers.

Reply:
![statetrigger-binding](https://user-images.githubusercontent.com/6755973/73851325-92b0a600-482d-11ea-9b5b-db51f5fd5134.gif)